Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift

```diff
@@ -327,6 +354,32 @@ final class StreamViewModel: ObservableObject {
             )
 
             displayMode = .list(listViewModel)
+        case .grid:
+            let secondaryVideoSources = sortedSources.filter { $0.id != selectedVideoSource.id }
```

Comment:
There are a lot of similarities to lines 162 to 186 above. Is it possible to group them into a single method?

Reply:
refactored, thanks